### PR TITLE
Cleanup around ViewImports

### PR DESCRIPTION
**Pull request by rynowak**
Added a default for the imports filename.

Deleted unused test files

### CODE REVIEW

File: src/Microsoft.AspNetCore.Mvc.Razor.Extensions/MvcRazorTemplateEngine.cs [hunk 1]
```diff
@@ -23,6 +23,7 @@ public class MvcRazorTemplateEngine : RazorTemplateEngine
             RazorProject project)
             : base(engine, project)
         {
+            Options.ImportsFileName = "_ViewImports.cshtml";
```
Comment (rynowak) [inline src/Microsoft.AspNetCore.Mvc.Razor.Extensions/MvcRazorTemplateEngine.cs]: This is going to be case for both Pages and Views

Comment (rynowak) [inline src/Microsoft.AspNetCore.Mvc.Razor.Extensions/MvcRazorTemplateEngine.cs]: Note that this is in the MVC extensions.

Comment (pranavkm) [inline src/Microsoft.AspNetCore.Mvc.Razor.Extensions/MvcRazorTemplateEngine.cs]: I guess this is going to break Mvc functionals. Could you get the PR for it ready?

Comment (rynowak) [inline src/Microsoft.AspNetCore.Mvc.Razor.Extensions/MvcRazorTemplateEngine.cs]: This should not break anything. The Page loader sets this value explicitly. In fact both places in MVC where this class are used set it.

I'm working on some other changes to MVC to streamline this.

**Comment by rynowak** [comment]
I don't think this is going to break anything because MVC sets this value. Getting this in first is going to help my MVC PR so I'm going to push it.

**Comment by rynowak** [comment]
3ec4e244db1d1398d7b0973b54ac607d1bd91227

**Pull request closed without merging**